custom_components/open_epaper_link/imagegen.py

```diff
@@ -175,12 +175,13 @@ def customimage(entity_id, service, hass):
             font_file = os.path.join(os.path.dirname(__file__), font)
             font = ImageFont.truetype(font_file, size)
             color = element.get('color', "black")
+            anchor = element.get('anchor', "lm")
             _LOGGER.debug("Got Multiline string: %s with delimiter: %s" % (element['value'],element["delimiter"]))
             lst = element['value'].replace("\n","").split(element["delimiter"])
             pos = element.get('start_y', pos_y + element.get('y_padding', 10))
             for elem in lst:
                 _LOGGER.debug("String: %s" % (elem))
-                d.text((element['x'], pos ), str(elem), fill=getIndexColor(color), font=font)
+                d.text((element['x'], pos ), str(elem), fill=getIndexColor(color), font=font, anchor=anchor))
```

Comment:
I think something is not quite right here :)

Reply:
Extra parenthesis at the end. Fixed at my end, how do I proceed to get int here again?